packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -353,12 +354,12 @@ function useRestoreFocus(scopeRef: RefObject<HTMLElement[]>, restoreFocus: boole
     };
 
     if (!contain) {
-      document.addEventListener('keydown', onKeyDown, false);
+      document.addEventListener('keydown', onKeyDown, true);
```

Comment:
This is because of Picker. Tabbing out of the menu should go to the element before the picker trigger, not the element before the list. So this event needs to run before the one in `useSelectableCollection`, and propagation stopped.